src/View/Helper/NumberHelper.php

```diff
@@ -152,7 +152,7 @@ public function format($number, array $options = [])
         $formatted = $this->_engine->format($number, $options);
         $options += ['escape' => true];
 
-        return $options['escape'] ? h($formatted) : $formatted;
+        return $options['escape'] ? (string)h($formatted) : $formatted;
```

Comment:
Why was this cast necessary? Seems wasteful

Reply:
Because h() being a god function.. Both string and array input output.

Reply:
`h()` can return a string or array, hence without the cast phpstan is unhappy.

Perhaps in 4.x we can make it just accept and return strings. I doubt passing array to it is commonly used.